index.html

```diff
@@ -3667,11 +3667,16 @@ <h3>Co-signed</h3>
               <td>Individual</td>
               <td>DevOps; SDE; Development; open-source community efforts</td>
             </tr>
-	    <tr>
+	          <tr>
```

Comment:
You messed up your formatting on this line, please fix it and push the new commit again.

Reply:
Thanks, @FraneCaleta, I fixed it!